plugin-flex-ts-template-v2/src/feature-library/enhanced-crm-container/custom-components/IFrameCRMContainer/IFrameWrapper/IFrameWrapper.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import * as Flex from '@twilio/flex-ui';
+
+import { wrapperStyle, frameStyle } from './IFrameWrapperStyles'
+
+const IFrameRefreshButtonStyledDiv = Flex.styled('div')`
+  align-items: center;
+  background-color: #5469D4;
+  border-radius: 50%;
+  bottom: 10px;
+  box-shadow: rgb(0 0 0 / 30%) 0px 1px 10px;
+  cursor: pointer;
+  display: flex;
+  height: 50px;
+  justify-content: center;
+  position: fixed;
+  right: 10px;
+  width: 50px;
+  z-index: 9999;
+`;
+
+export interface Props {
+  thisTask: Flex.ITask, // task assigned to iframe
+  task: Flex.ITask // task in Context
+  baseUrl: string
+}
+
+export class IFrameWrapper extends Component<Props> {

```

Comment:
Turning this into a function could improve readability and allow using some new hooks.